workflows_api/runtime/src/schemas.py

```diff
@@ -262,6 +268,8 @@ def check_sample_files(cls, values):
             )
         return values
 
+    class Config:
+        extra = Extra.allow
```

Comment:
does this not get inherited from `Dataset` ?

Reply:
makes sense. removing it.

Reply:
makes sense. Removing it!